genesyscloud/outbound_messagingcampaign/resource_genesyscloud_outbound_messagingcampaign_schema.go

```diff
@@ -197,6 +216,22 @@ func ResourceOutboundMessagingCampaign() *schema.Resource {
 					return false
 				},
 			},
+			`callable_time_set_id`: {
+				Description: `The callable time set for this messaging campaign.`,
+				Optional:    true,
+				Type:        schema.TypeString,
+			},
+			`contact_list_id`: {
+				Description: `The contact list that this messaging campaign will send messages for.`,
+				Required:    true,
+				Type:        schema.TypeString,
+			},
+			`dnc_list_ids`: {
+				Description: `The dnc lists to check before sending a message for this messaging campaign.`,
+				Optional:    true,
+				Type:        schema.TypeList,
```

Comment:
This should not be changed to TypeList. It was changed to TypeSet to address a bug in this PR - https://github.com/MyPureCloud/terraform-provider-genesyscloud/pull/1446